modules/mono/glue/GodotSharp/GodotSharp/Compat.cs

```diff
@@ -127,3 +127,23 @@ public void CreateAction(string name, MergeMode mergeMode)
         CreateAction(name, mergeMode, backwardUndoOps: false);
     }
 }
+
+partial struct Basis
+{
+    /// <inheritdoc cref="LookingAt(Vector3, Vector3?, bool)"/>
```

Comment:
We've been using `Nullable{T}` syntax in documentation to refer to nullable value types, so I'd like to keep it that way for consistency.

```suggestion
    /// <inheritdoc cref="LookingAt(Vector3, Nullable{Vector3}, bool)"/>
```